SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -345,15 +210,17 @@ void InterestFactory::AddPlayerControllerActorInterest(Interest& OutInterest, co
 {
 	QueryConstraint LevelConstraint = CreateLevelConstraints(InActor);
 
+	// Always relevant and always interested
 	AddSystemQuery(OutInterest, InActor, InInfo, LevelConstraint);
 
-	AddUserDefinedQueries(OutInterest, InActor, LevelConstraint);
-
-	// If net cull distance frequency queries are enabled, build and add those separately as they have to be built each time.
-	// They are added as separate queries for the same reason- different frequencies.
-	if (GetDefault<USpatialGDKSettings>()->bEnableNetCullDistanceFrequency)
+	if (HasUserDefinedConstraint(InActor))
```

Comment:
`bUseNetCullDistanceSquaredForCheckoutRadius` errrrrrrrrrrrr..........

Reply:
you have a way with words

fixed